source/content/drupal-9.md

```diff
@@ -13,19 +13,17 @@ Drupal 9 is, according to [drupal.org](https://www.drupal.org/docs/understanding
 
 Drupal 9 updates Drupal’s underlying dependencies like [Symfony 4.4](https://symfony.com/releases/4.4) and [Twig 2](https://twig.symfony.com/doc/2.x/index.html), removes several deprecated API functions in favor of better options, and allows everyone running Drupal 8.8+ an easy upgrade path to Drupal 9 and beyond.
```

Comment:
While Drupal 9.0 was all about removing deprecations, Drupal 9.1 has "new cutting-edge features" and will be supported beyond the Nov 2021 Drupal 8 end-of-life. So I think this intro should be less measured and more celebratory of Drupal 9. Maybe take some cues from https://www.drupal.org/about/9 or https://www.drupal.org/about/media-kit/drupal-9  -- maybe @mmlc or @stevector can help too?